packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/utils/regex.ts

```diff
@@ -1,6 +1,6 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-export const telOrMailRegex = /^(mailto:|tel:)[\w%+.-@]+$/;
+export const telOrMailRegex = /^(mailto:|tel:)([\w %+.@\-]+)$/i;
 export const urlRegex = /[^\s#$./?].\S*\.[a-z]{2,}$/i;
 export const urlRegexWithHttps = /^(https?:\/\/)[^\s#$./?].\S*\.[a-z]{2,}$/i;
```

Comment:
Is this still used somewhere? If not it can be removed.

Reply:
good point! I removed them and changed the addHttps such that it uses the same logic as the validation